Ecommerce/app/controllers/welcome_controller.rb

```diff
@@ -2,6 +2,6 @@ class WelcomeController < ApplicationController
   before_action :authenticate_user!
 
   def index
-      @items = Item.all
-  end
+   @q = Item.ransack(params[:q])
```

Comment:
if we want to show items, why we are not going into items controller? why we are hitting welcome controller?